### ISSUE TITLE
Example of like within PGB app

### ISSUE THREAD

**Issue by vespino** [Wizcorp/phonegap-facebook-plugin#1036]
Is it possible to post an example of how to use this plugin and especially how to make it possible to like a page from within an app?

Thanks!

**Comment by DeclanMurphyCit** [comment]
I don't think that it's possible to do this yet. Facebook themselves are the ones not allowing it.
